WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -296,8 +306,8 @@ private extension WooShippingCreateLabelsView {
             static let total = NSLocalizedString("wooShipping.createLabels.bottomSheet.total",
                                                         value: "Total",
                                                         comment: "Label for row showing the total for shipment costs on the shipping label creation screen")
-            static let markComplete = NSLocalizedString("wooShipping.createLabels.bottomSheet.markComplete",
-                                                        value: "Mark this order complete and notify the customer",
+            static let markComplete = NSLocalizedString("wooShipping.createLabels.bottomSheet.afterPurchaseMarkComplete",
+                                                        value: "After purchasing a label, mark this order as complete and notify the customer",
```

Comment:
Let's not change this without consulting with design. The designs include some intentional deviations from the web — IIRC this string was explicitly changed for mobile — to better fit mobile screen sizes, and in this case there's also localization to consider (since some languages will make this string even longer).

Reply:
Should we then also change it back on Android to "Mark this order complete..."?
Since Android uses "After purchasing a label..."

Reply:
Ah, I didn't realize this was also aligning with Android, as well as web. @atorresveiga what was the reasoning behind using the web string on Android?

Reply:
> what was the reasoning behind using the web string on Android?


It was proposed by design p1732107701725609/1732055886.685969-slack-C02KUCFCSFP

Reply:
Oh, it's explicitly part of this task 🤦‍♀️ I missed that somehow, thanks 👍